hack/e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -e
+
+if [[ -z "$RESOURCEGROUP" ]]; then
+    RESOURCEGROUP=$(awk '/^    resourceGroup:/ { print $2 }' <_data/containerservice.yaml)
+fi
+
+echo "Running end user e2e tests"
+# Login as osadmin to simulate a regural user
+password=$(hack/config.sh get-config $RESOURCEGROUP | jq -r .config.adminPasswd)
+fqdn=$(hack/config.sh get-config $RESOURCEGROUP | jq -r .properties.fqdn)
+oc login $fqdn --username osadmin --password $password --insecure-skip-tls-verify=true
```

Comment:
We need to think what we gonna do when we ditch this one. @cjryan already raised the same request for their testing (need of test users). 
Im thinking to maybe add htpassws as identity provider which we can use in cases like this...

Reply:
Unfortunately it's impossible to `oc login` using AAD, otherwise we could use a bot account in Azure to do the login. We should probably raise an issue in Origin about adding AAD support in `oc login`.

Reply:
For clarity, in the future if we want to get rid of htpasswd altogether and we still cannot use AAD via the cli, we could create a user here, grant it project admin, and pass system:admin in the test with delegation to user. Dirty, but could work. I hope we can figure out how to use AAD.